Common/Models/Users/TraktUser.swift

```diff
@@ -16,13 +16,15 @@ public struct User: Codable {
     public let name: String?
     public let isVIP: Bool?
     public let isVIPEP: Bool?
+    //    public let ids: ID
```

Comment:
These changes should be reverted, I just merged a PR that added support for this.